lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -354,7 +366,36 @@ def send_bulk(data)
     begin
       response = client.bulk body: data
       if response['errors']
-        log.error "Could not push log to Elasticsearch: #{response}"
+        errors = Hash.new(0)
+        errors_bad = 0
+        errors_unrecognized = 0
+        # Count up the individual error types returned for each item
+        response['items'].each do |item|
+          begin
+            if [429, 500].include?(item['create']['status'])
```

Comment:
Need some sort of condition before this so that successful operations will be skipped https://www.elastic.co/guide/en/elasticsearch/guide/current/bulk.html
e.g.
```ruby
if [200, 201].include?(item['create']['status'])
  successful_ops += 1
elsif [429, 500].include?....
```